search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -170,6 +171,52 @@ private IndexDocument crateSampleIndexDocument(String indexName, String jsonFile
         return new IndexDocument(eventConsumptionAttributes, jsonNode);
     }
 
+    private void addDocumentsToIndex(String... files) throws InterruptedException {
+        Arrays.asList(files)
```

Comment:
Stream.of(files) is marginal faster, in most cases... and you don't need a list/array